codice-itest-impl/src/main/java/org/codice/itest/result/TestResultFactoryImpl.java

```diff
@@ -65,6 +66,13 @@ public TestResult error(String testName, Throwable throwable, Instant startTime,
         return new ErrorTestResultImpl(runId, testName, throwable, startTime, endTime);
     }
 
+    @Override
+    public TestResult notExecuted(String testName) {
+        MDC.put(STACK_TRACE, "Not Executed");
```

Comment:
:pencil2: I don't think this is necessary as it introduces a new special value for STACK_TRACE. Can it just be omitted?